<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCPD UDP Dev Gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cmd Host &amp; Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_cmdHost">
           <property name="text">
            <string>192.168.168.121</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_cmdPort">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>54321</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data Rcv Port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spin_dataPort">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>54322</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="gb_packets">
          <property name="title">
           <string>Packets</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Packets Directory</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_packetsDir">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pb_openPacketsDir">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Open</string>
               </property>
               <property name="icon">
                <iconset theme="folder-open">
                 <normaloff>../../../mesytec-usb-utils/src</normaloff>../../../mesytec-usb-utils/src</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_browsePacketsDir">
               <property name="text">
                <string>Browse</string>
               </property>
               <property name="icon">
                <iconset theme="folder-open">
                 <normaloff>../../../mesytec-usb-utils/src</normaloff>../../../mesytec-usb-utils/src</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QTableView" name="filesView"/>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="pb_newPacket">
               <property name="text">
                <string>New Packet</string>
               </property>
               <property name="icon">
                <iconset theme="document-new">
                 <normaloff>../../../mesytec-usb-utils/src</normaloff>../../../mesytec-usb-utils/src</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_removePacket">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Remove Packet</string>
               </property>
               <property name="icon">
                <iconset theme="user-trash">
                 <normaloff>../../../mesytec-usb-utils/src</normaloff>../../../mesytec-usb-utils/src</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="gb_cmd">
          <property name="title">
           <string>Cmd/Log</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="gb_data">
          <property name="title">
           <string>Data</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
